.github/workflows/deployment.yml

```diff
@@ -292,7 +289,9 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: JabRef-${{ matrix.os }}
-          path: jabgui/build/distribution
+          path: |
+            jabgui/build/distribution
+            jabkit/build/distribution
```

Comment:
jabkit is another 300mb extra, this will kill our space because I don't know how to extend the disk space in azure

Reply:
Context is "- name: Upload to GitHub workflow artifacts store", which is not our build server.

But I will check the others and only upload the "main" build